history.html

```diff
@@ -0,0 +1,269 @@
+        href="https://github.com/astropy/astropy/wiki/CfAMeeting2011%20“CfA%20Astropy%20meeting”">Harvard
+        Center for Astrophysics</a>. The <a
+        href="https://github.com/astropy/astropy/wiki/CfAMeeting2011#participants%20“CfA%20meeting%20participants”">list
+        of attendees</a> at that meeting underscores what has been essential to the
+        launch, growth, and continued development of Astropy: it was a mix of
+        graduate students, postdocs, scientists and professional software developers.
+        The attendees with permanent positions were willing to contribute both code
+        and their time to the project; Space Telescope Science Institute
+        (STScI) additionally contributed substantial staff time to the project. The
+        early-career attendees had either already devoted substantial time to code
+        development, would do so over the ensuing years, or both.</p>
+
+        <p>The foundation of Astropy’s subsequent success was the combination of
+        institutional resources, a deliberate effort to include and foster the growth
+        of a broad community of contributors, the rapid growth of GitHub and the
+        surrounding ecosystem of open source development tools, and a willingness of
+        early-career professionals to contribute code to an open community project.
+        The initial release for users of the core astropy package, version 0.2 on
+        February 19, 2013, was less than 18 months after the CfA meeting and already
+        contained many of the core subpackages that are part of the package today.
+        That was possible only because some of the code already existed in a form
+        that could be adapted to Astropy. Major pieces had been written by staff at
+        STScI that were contributed by the Institute to the project. There were also
+        large contributions from early participants who were graduate students and
+        postdocs.</p>
+
+        <p>By the time of the first stable release in 2013, the number of contributors
+        to the code base was over 20, including several people who were not involved
+        in the initial meeting. Though most of the lines of code at that point had
+        been written by a handful of people, the effort they put into welcoming and
+        supporting new contributors was just as important and is not easily captured
+        in a single number. The project made an effort early on to provide prompt,
+        constructive, and welcoming feedback to new contributors. The promptness was
+        a key factor in encouraging early contributors and was possible in part
+        because STScI devoted substantial staff time to the project with the explicit
+        intent of growing the community of contributors.</p>
+
+        <p>One of the Project’s first efforts to formally recruit early career
+        scientists was participation in the [Google’s Summer of Code (GSoC) program]
```

Comment:
Why the square brackets?
```suggestion
        scientists was participation in the Google’s Summer of Code (GSoC) program,
```

Reply:
That was left over from conversion from markdown to html. Likely it means the link isn't in there either.

Reply:
Fixed, thanks!